app/poros/user_github_repos.rb

```diff
@@ -0,0 +1,9 @@
+class UserGithubRepos
+  def repos
+    return @repo_data if @repo_data
+    service = GithubService.new
+    @repo_data = service.get_repos.map do |repo_data|
```

Comment:
Instead of the `return`... `if` guard clause, I think memoization could also be used here. Something like:
```
  service = GithubService.new
  @repo_data ||= service.get_repos.map do |repo_data|
 ```